codespell_lib/data/dictionary.txt

```diff
@@ -1303,6 +1303,9 @@ alignrigh->alignright
 alikes->alike, likes,
 aline->align, a line, line, saline,
 alined->aligned
+aling->align
+alinged->aligned
+alinging->aligning
```

Comment:
Could also add `alings->aligns` ?

Hmmm, I wonder if e.g. `aling` should also suggest `sling` as a possible correction? And maybe `ailing`? :man_shrugging: Oooh, and `along` is only a single-key different too...

Reply:
@lurch done @ 04eec2b4f5c5198bc97e08a230b096a934dd6dcb